web/packages/teleport/package.json

```diff
@@ -44,7 +44,7 @@
   "devDependencies": {
     "@gravitational/build": "workspace:*",
     "@gravitational/teleport": "workspace:*",
-    "babel-plugin-transform-import-meta": "^2.2.0",
+    "babel-plugin-transform-import-meta": "^2.2.1",
```

Comment:
Still waiting for a new version to be released which would include the fix for the license name. https://github.com/javiertury/babel-plugin-transform-import-meta/pull/23